modules/ROOT/pages/custom-policy-set-authentication-example.adoc

```diff
@@ -0,0 +1,199 @@
+
+[[add-server_in-settings]]
+=== Add a Server in the settings.xml File
+To have access to the repository, ProTeams needs to add the corresponding server in their Maven settings file:
+
+[source,XML,linenums]
+----
+<servers>
+      <server>
+         <id>releases-ee</id>
+         <username>your_username</username>
+         <password>your_password</password>
+       </server>
+</servers>
+----
+
+[[add_dependencies_in_pom]]
+=== Add Dependencies in the pom.xml File
+
+After adding a new server in their Maven `settings.xml` file, ProTeams then adds dependencies such as the Event Authentication and the HTTP Policy Transform extensions to the `pom.xml` file, for the policy to perform specific actions:
+
+
+[source,XML,linenums]
+----
+    <dependencies>
+        <dependency>
+            <groupId>com.mulesoft.anypoint</groupId>
+            <artifactId>mule-event-authentication-extension</artifactId>
+            <version>${authenticationExtensionVersion}</version>
+            <classifier>mule-plugin</classifier>
+        </dependency>
+        <dependency>
+            <groupId>com.mulesoft.anypoint</groupId>
+            <artifactId>mule-http-policy-transform-extension</artifactId>
+            <version>${httpPolicyTransformVersion}</version>
+            <classifier>mule-plugin</classifier>
+            <exclusions>
+                <exclusion>
+                    <groupId>org.mule.connectors</groupId>
+                    <artifactId>mule-http-connector</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
+    </dependencies>
+----
+
+[[add_conditions_in_template]]
+=== Add Conditions in the template.xml File
+
+ProTeams now needs to add functionality logic to the `template.xml file` to enable the policy to perform tasks, such as the ID mapping for ProTeams.
+
+Because ProTeams value mapping is not available in a Mule extension, ProTeams must create and add their own custom module.
+
+[[upload_to_exchange]]
+== Uploading to Exchange
+
+To upload the custom policy to Exchange, ProTeams must verify that their `pom.xml` file is correctly configured for their organization.
+
+=== Task Prerequisites
+
+Before uploading their policy to Exchange, ProTeams must verify that:
+
+* A new server is created in their Maven `settings.xml` file, with a unique ID, and username and password of the Exchange credentials for their organization.
+* The URL matches their region. 
++
+*Warning:* When you perform similar actions to create your own custom policy, note that the template uses the URL that corresponds to the US region. If your account does not belong to this region, change this value.
+* The repository ID matches the one from the Maven settings. 
++
+*Tip:* You can define an alias to that repository using the name tag.
+
+//Nacho haven't they already added a server in the previous section? I think we should remove this step.
```

Comment:
Yes, they added a server previously, but with a fixed ID (as it needs to match the ID inside the pom.xml of the example). Here they can call it whatever they want, as long as it matches the pom.xml distributionManagement element.
Maybe it is kind of redundant and the code snippet can be skipped, I have no preference here.

Reply:
OK, I'll remove it then. It is confusing this way. Done! Thanks!